packages/j/juliaup/package.yml

```diff
@@ -0,0 +1,24 @@
+name       : juliaup
+version    : 1.11.22
+release    : 1
+source     :
+    - https://github.com/JuliaLang/juliaup/archive/refs/tags/v1.11.22.tar.gz : 8a0b4540262a8bea9e61a4f287ced32ec4ad2ac4cb528a1b473eea6f139256df
+license    : MIT
+component  : programming.tools
+summary    : Julia installer and version multiplexer 
+description: |
+    A cross-platform installer for the Julia programming language.
+replaces   : julia
+networking : yes
+builddeps  :
+    - cargo
+setup      : |
+    # It seems like the `built` crate is having trouble fetching system
+    # information inside solbuild's container environment
+    sed -i 's|built::write_built_file().expect("Failed to acquire build-time information");||' build.rs
+    sed -i 's|include!(concat!(env!("OUT_DIR"), "/built.rs"))|pub const PKG_VERSION: \&str = "%version%"|' src/lib.rs
+build      : |
+    cargo build %JOBS% --release --bin juliaup --bin julialauncher --features binjulialauncher
```

Comment:
Need to add the `--frozen` flag to make the build reproducible by only downloading pinned dependencies (in combination with `cargo fetch --locked`).